cypress/pages/workflow.ts

```diff
@@ -26,6 +26,8 @@ export class WorkflowPage extends BasePage {
 		canvasNodes: () => cy.getByTestId('canvas-node'),
 		canvasNodeByName: (nodeName: string) =>
 			this.getters.canvasNodes().filter(`:contains(${nodeName})`),
+		nodeIssuesByName: (nodeName: string) =>
+			this.getters.canvasNodes().filter(`:contains(${nodeName})`).findChildByTestId('node-issues'),
```

Comment:
This could lead to flakiness. `.filter` is not retryable so if the canvas nodes are found but none contain the specified name(because didn't fully render yet), then `.filter` will return an empty set and `.findChildByTestId` will not be able to find any child since there's no parent to begin with. To get around this we could add `.should` after the filter to make sure the last command(filter) is retried. 
```
this.getters.canvasNodes()
     .filter(`:contains(${nodeName})`)
     .should('have.length.greaterThan', 0)
     .findChildByTestId('node-issues')
 ```

Reply:
arlight thanks